test/widgets/message_list_test.dart

```diff
@@ -981,4 +981,70 @@ void main() {
       });
     });
   });
+
+  group('TypingStatus', () {
```

Comment:
nit: this name looks like an identifier rather than normal English (being in PascalCase), so it should be the actual identifier
```suggestion
  group('TypingStatusWidget', () {
```